woosmapgeofencing/src/main/java/com/webgeoservices/woosmapgeofencing/Woosmap.java

```diff
@@ -468,4 +482,66 @@ public void run() {
             }
         });
     }
+
+    public void stopTracking() {
+        WoosmapSettings.trackingEnable = false;
+        this.locationManager.removeLocationUpdates();
+    }
+
+    public String loadJSONFromAsset(String fileName) {
+        String json = null;
+        try {
+            InputStream is = this.context.getAssets().open(fileName +".json");
+            int size = is.available();
+            byte[] buffer = new byte[size];
+            is.read(buffer);
+            is.close();
+            json = new String(buffer, "UTF-8");
+        } catch (IOException ex) {
+            ex.printStackTrace();
+            return null;
+        }
+        return json;
+    }
+
+    public void startTracking(String profile) {
+
+        try {
+            JSONObject obj = new JSONObject(loadJSONFromAsset(profile));
+            WoosmapSettings.trackingEnable = obj.getBoolean( "trackingEnable" );
+            WoosmapSettings.modeHighFrequencyLocation = obj.getBoolean( "modeHighFrequencyLocation" );
```

Comment:
`foregroundLocationServiceEnable` a été oublié 

```suggestion
            WoosmapSettings.foregroundLocationServiceEnable = obj.getBoolean( "foregroundLocationServiceEnable" );
            WoosmapSettings.modeHighFrequencyLocation = obj.getBoolean( "modeHighFrequencyLocation" );
```